pages/signin.tsx

```diff
@@ -1,67 +1,99 @@
-          <label className='mb-2 block text-sm font-bold text-gray-700' htmlFor='email'>
-            Username
-          </label>
-          <input
-            className='focus:shadow-outline w-full appearance-none rounded border py-2 px-3 leading-tight text-gray-700 shadow focus:outline-none'
-            id='email'
-            type='email'
-            placeholder='E-Mail'
-            value={email}
-            onChange={e => setEmail(e.target.value)}
-          />
-        </div>
-        <div className='w-50 mt-5'>
-          <label className='mb-2 block text-sm font-bold text-gray-700' htmlFor='password'>
-            Password
-          </label>
-          <input
-            className='focus:shadow-outline w-full appearance-none rounded border py-2 px-3 leading-tight text-gray-700 shadow focus:outline-none'
-            id='password'
-            type='password'
-            placeholder='Password'
-            value={password}
-            onChange={e => setPassword(e.target.value)}
-          />
-        </div>
-        <button
-          className='focus:shadow-outline mt-5 rounded bg-blue-500 py-2 px-4 font-bold text-white hover:bg-blue-700 focus:outline-none'
-          type='submit'
-        >
-          Sign In
-        </button>
-      </form>
-    </div>
-  );
+				<div className='flex h-screen  w-[100%] flex-col items-center justify-center bg-gray-100 lg:w-[50%]'>
+					{/* Now you can decide different login ways - primarily I am giving google and twiiter for more add it yourself. */}
+					<div className='w-full  bg-white p-4 md:shadow-lg lg:h-[500px] lg:w-[400px]'>
+						<h1 className='p-2 text-3xl font-semibold'>Login ⚡</h1>
+
+						<form
+							className='mt-4'
+							onSubmit={e => {
+								e.preventDefault();
+								handleSignIn();
+							}}
+						>
+							<div className='flex w-[90%] flex-col space-y-2'>
+								<p>Email</p>
+								<input
+									type='text'
+									placeholder='example@gmail.com'
+									required
+									className='input-field outline-none'
+									value={email}
+									onChange={e => setEmail(e.target.value)}
+								/>
+							</div>
+							<div className='flex w-[90%] flex-col space-y-2 '>
+								<p>Password</p>
+								<input
+									type='password'
+									required
+									className='input-field outline-none'
+									placeholder='*****'
+									value={password}
+									onChange={e => setPassword(e.target.value)}
+								/>
+							</div>
+						</form>
+						<button
+							type='submit'
+							className='my-4 flex  w-[90%] cursor-pointer items-center  justify-center rounded-md bg-blue-600 py-2 font-semibold text-white hover:bg-blue-500'
+							onClick={handleSignIn}
+						>
+							Login
+						</button>
+						<p className='py-2 font-semibold'>
+							Not registered yet ?
+							<span
+								className='cursor-pointer font-bold text-blue-600 hover:underline'
+								onClick={() => router.push('/signup')}
+							>
+								Register here
```

Comment:
add a space before "Register here" and make sure it shows up on the UI

also, we should say "Sign up" instead of "Register"

Reply:
Actually {" "} this was giving up the space which we removed earlier. 😅